cmd/fabric8-jenkins-idler/idler.go

```diff
@@ -52,33 +53,33 @@ func (idler *Idler) Run() {
 	defer cancel()
 
 	// Create Idler controller
-	controller := openshift.NewOpenShiftController(
+	controller := openshift.NewControllerImpl(
```

Comment:
Do you really need 'Impl'?

Reply:
Nope, but if the struct is `ControllerImpl` then it makes sense to have a function named `NewControllerImpl`

Reply:
Let me rephrase, why do you need to call it ControllerImpl? Don't you get away with 'controller'?